services/pypi/pypi-helpers.js

```diff
@@ -50,7 +50,7 @@ function getLicenses(packageData) {
   const {
     info: { license },
   } = packageData
-  if (license) {
+  if (license && license.length < 40) {
```

Comment:
As noted in https://github.com/badges/shields/issues/8689#issuecomment-1341526309 we're going to have to make some kind of assumption here. The longest SPDX identifier in https://raw.githubusercontent.com/jslicense/spdx-license-ids/main/index.json is 36 characters so this seems like a reasonable-ish threshold.

My hunch is that the license meta-data field is more commonly used than the classifiers, so I think switching to prefer the classifiers in all cases is probably going to break more badges out in the wild than doing this.

There isn't really a perfect solution here.

Reply:
Agreed with the approach. I do think it might be worth adding an inline comment explaining this though so that `40` doesn't appear like a magic number to readers of the code.

I recognize that one could use git blame info to backtrack to this PR and explanation, but I'm a fan of having things be as self-explanatory as possible within the code